course_discovery/apps/publisher/mixins.py

```
@@ -65,20 +72,17 @@ def check_roles_access(user):
     return False
 
 
-def check_course_organization_permission(user, course):
+def check_course_organization_permission(user, course, permission):
     """ Return True if user has view permission on organization. """
+    if not permission:
+        return False
```

Per my comment above:  I think this should raise an exception rather than just return false.  You should not be allowed to check the permission if no permission is passed